### PR TITLE
influxdb: Envars for flush options

### PR DESCRIPTION
**Pull request by codebien**
Added missing options for adjusting the flush rate.

### CODE REVIEW

File: src/data/markdown/translated-guides/es/04 Results visualization/06 InfluxDB - Grafana.md [hunk 1]
```diff
@@ -134,6 +134,9 @@ Al subir los resultados de k6 a InfluxDB (`k6 run --out influxdb=`), puede confi
...
 | `K6_INFLUXDB_TAGS_AS_FIELDS` |  Un string separado por comas para establecer las métricas de k6 como [campos no indexables](https://docs.influxdata.com/influxdb/v1.8/concepts/glossary/#field) (en lugar de etiquetas). Se puede especificar un tipo opcional usando `:type` como en `vu:int` que hará que el campo sea entero. Los tipos de campo posibles son `int`, `bool`, `float` y `string`, que es el predeterminado. Ejemplo: `vu:int,iter:int,url:string,event_time:int` |         |
+| K6_INFLUXDB_PUSH_INTERVAL     | The flush's frequency of the `k6` metrics. | 1s |
+| K6_INFLUXDB_CONCURRENT_WRITES | Number of concurrent requests for flushing data. It is useful to not block the process when a request takes more than the expected time (more than flush interval). | 4 |
+
```
Comment (MStoykov) [inline src/data/markdown/translated-guides/es/04 Results visualization/06 InfluxDB - Grafana.md:139]: Is this how it should be done @ppcano ? us adding the English and then waiting for someone to translate it to Spanish?

**Review by MStoykov — APPROVED** [review]

File: src/data/markdown/translated-guides/es/04 Results visualization/06 InfluxDB - Grafana.md (hunk 1 above)
Comment (codebien) [reply src/data/markdown/translated-guides/es/04 Results visualization/06 InfluxDB - Grafana.md]: @ppcano can I merge it or do you plan to provide the translation directly here?

Comment (ppcano) [reply src/data/markdown/translated-guides/es/04 Results visualization/06 InfluxDB - Grafana.md]: Please, merge it. Let's do not get blocked with the translations.

**Review by yorugac — COMMENTED** [review]
The formatting is off:
Screenshot from 2021-11-05 10-55-09

**Review by yorugac — APPROVED** [review]

**Comment by codebien** [comment]
@ppcano ready, when you want this can be merged.

**Pull request merged**